Remove a callback check from the `agent_info_sync` test

## Description

Before version 4.3.0, all the chunks that were being sent to the master could be found reported in the `cluster.log` file, using the `debug2` mode. However, this was removed in this [PR](https://github.com/wazuh/wazuh/pull/10807/files), as this created much noise and consumed a lot of space. 

 This issue is being opened in order to remove the part of the `agent_info_sync` test responsible for looking for a specific chunk among all the logs present in the `cluster.log` file. 
https://github.com/wazuh/wazuh-qa/blob/158ab10a0dbbf0c9dcbd4cc2c718346bc8ba9c49/tests/system/test_cluster/test_agent_info_sync/test_agent_info_sync.py#L72

This is causing the test to fail: 
```
python3.9 -m pytest tests/system/test_cluster/test_agent_info_sync/ -sxvv
/home/yanazaeva/.local/lib/python3.9/site-packages/_testinfra_renamed.py:5: DeprecationWarning: testinfra package has been renamed to pytest-testinfra. Please `pip install pytest-testinfra` and `pip uninstall testinfra` and update your package requirements to avoid this message
  warnings.warn((
==================================================================================== test session starts =====================================================================================
platform linux -- Python 3.9.5, pytest-6.2.5, py-1.10.0, pluggy-1.0.0 -- /usr/bin/python3.9
cachedir: .pytest_cache
rootdir: /home/yanazaeva/git/wazuh-qa
plugins: aiohttp-0.3.0, trio-0.7.0, testinfra-6.4.0, asyncio-0.16.0, cov-2.12.0, testinfra-6.0.0
collected 2 items                                                                                                                                                                            

tests/system/test_cluster/test_agent_info_sync/test_agent_info_sync.py::test_agent_info_sync 2021-11-26 12:01:51,397 - wazuh_testing - DEBUG - Add new file composer process for wazuh-master and path: /var/ossec/logs/cluster.log
2021-11-26 12:01:51,399 - wazuh_testing - DEBUG - Add new file monitor process for wazuh-master and path: /var/ossec/logs/cluster.log
2021-11-26 12:01:51,399 - wazuh_testing - DEBUG - Starting file composer for wazuh-master and path: /var/ossec/logs/cluster.log. Composite file in /home/yanazaeva/git/wazuh-qa/tests/system/test_cluster/test_agent_info_sync/tmp/wazuh-master_cluster.log.tmp
2021-11-26 12:01:51,400 - wazuh_testing - DEBUG - Starting QueueMonitor for wazuh-master and message: .*test_label.*
2021-11-26 12:02:51,554 - wazuh_testing - DEBUG - Finishing QueueMonitor for wazuh-master and message: .*test_label.*
2021-11-26 12:02:52,462 - wazuh_testing - DEBUG - Cleaning temporal files...
2021-11-26 12:02:52,462 - wazuh_testing - DEBUG - Checking results...
FAILED


=============================================================================================== FAILURES ================================================================================================
_________________________________________________________________________________________ test_agent_info_sync __________________________________________________________________________________________

clean_cluster_logs = None, remove_labels = None

    def test_agent_info_sync(clean_cluster_logs, remove_labels):
        """Check agent agent-info synchronization works as expected.
    
        This test will wait for the expected agent-info messages declared in data/messages.yml. Additionally, it will
        ensure agent-info synchronization is working by modifying one agent."""
    
        # Add a label to one of the agents and restart it to apply the change
        host_manager.add_block_to_file(host=modified_agent, path=f'{WAZUH_PATH}/etc/ossec.conf',
                                       after='</client>', before='<client_buffer>',
                                       replace=f'<labels><label key="{label}">value</label></labels>')
        host_manager.get_host(modified_agent).ansible('command', f'service wazuh-agent restart', check=False)
    
        # Run the callback checks for the Master and Worker nodes
>       HostMonitor(inventory_path=inventory_path, messages_path=messages_path, tmp_path=tmp_path).run()

tests/system/test_cluster/test_agent_info_sync/test_agent_info_sync.py:72: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
../../.local/lib/python3.9/site-packages/wazuh_testing/tools/monitoring.py:903: in run
    self.check_result()
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <wazuh_testing.tools.monitoring.HostMonitor object at 0x7fdc4fc22f70>

    def check_result(self):
        """Check if a TimeoutError occurred."""
        logger.debug(f'Checking results...')
        while not self._queue.empty():
            result = self._queue.get(block=True)
            for host, msg in result.items():
                if isinstance(msg, TimeoutError):
>                   raise msg
E                   TimeoutError: Did not found the expected callback in wazuh-master: .*test_label.*

../../.local/lib/python3.9/site-packages/wazuh_testing/tools/monitoring.py:987: TimeoutError
------------------------------------------------------------------------------------------ Captured log setup -------------------------------------------------------------------------------------------
DEBUG    testinfra:base.py:286 RUN CommandResult(command=b'ansible --tree /tmp/tmpq2_lwo3_ -i /home/yanazaeva/git/wazuh-qa/tests/system/provisioning/basic_cluster/inventory.yml -m copy --args \'dest=/var/ossec/logs/cluster.log content=\'"\'"\'\'"\'"\' force=yes\' wazuh-master', exit_status=0, stdout=b'wazuh-master | CHANGED => {\n    "changed": true,\n    "checksum": "da39a3ee5e6b4b0d3255bfef95601890afd80709",\n    "dest": "/var/ossec/logs/cluster.log",\n    "gid": 999,\n    "group": "wazuh",\n    "md5sum": "d41d8cd98f00b204e9800998ecf8427e",\n    "mode": "0660",\n    "owner": "wazuh",\n    "size": 0,\n    "src": "/root/.ansible/tmp/ansible-tmp-1637924445.9513495-212094640354081/source",\n    "state": "file",\n    "uid": 999\n}\n', stderr=None)
INFO     testinfra:ansible.py:71 RUN Ansible('copy', "dest=/var/ossec/logs/cluster.log content='' force=yes", {'check': False, 'become': False}): {'changed': True,
 'checksum': 'da39a3ee5e6b4b0d3255bfef95601890afd80709',
 'dest': '/var/ossec/logs/cluster.log',
 'gid': 999,
 'group': 'wazuh',
 'md5sum': 'd41d8cd98f00b204e9800998ecf8427e',
 'mode': '0660',
 'owner': 'wazuh',
 'size': 0,
 'src': '/root/.ansible/tmp/ansible-tmp-1637924445.9513495-212094640354081/source',
 'state': 'file',
 'uid': 999}
DEBUG    testinfra:base.py:286 RUN CommandResult(command=b'ansible --tree /tmp/tmp8_bwhcwy -i /home/yanazaeva/git/wazuh-qa/tests/system/provisioning/basic_cluster/inventory.yml -m copy --args \'dest=/var/ossec/logs/cluster.log content=\'"\'"\'\'"\'"\' force=yes\' wazuh-worker1', exit_status=0, stdout=b'wazuh-worker1 | CHANGED => {\n    "changed": true,\n    "checksum": "da39a3ee5e6b4b0d3255bfef95601890afd80709",\n    "dest": "/var/ossec/logs/cluster.log",\n    "gid": 999,\n    "group": "wazuh",\n    "md5sum": "d41d8cd98f00b204e9800998ecf8427e",\n    "mode": "0660",\n    "owner": "wazuh",\n    "size": 0,\n    "src": "/root/.ansible/tmp/ansible-tmp-1637924447.7851708-216240098106218/source",\n    "state": "file",\n    "uid": 999\n}\n', stderr=None)
INFO     testinfra:ansible.py:71 RUN Ansible('copy', "dest=/var/ossec/logs/cluster.log content='' force=yes", {'check': False, 'become': False}): {'changed': True,
 'checksum': 'da39a3ee5e6b4b0d3255bfef95601890afd80709',
 'dest': '/var/ossec/logs/cluster.log',
 'gid': 999,
 'group': 'wazuh',
 'md5sum': 'd41d8cd98f00b204e9800998ecf8427e',
 'mode': '0660',
 'owner': 'wazuh',
 'size': 0,
 'src': '/root/.ansible/tmp/ansible-tmp-1637924447.7851708-216240098106218/source',
 'state': 'file',
 'uid': 999}
DEBUG    testinfra:base.py:286 RUN CommandResult(command=b'ansible --tree /tmp/tmp9s3x9po4 -i /home/yanazaeva/git/wazuh-qa/tests/system/provisioning/basic_cluster/inventory.yml -m copy --args \'dest=/var/ossec/logs/cluster.log content=\'"\'"\'\'"\'"\' force=yes\' wazuh-worker2', exit_status=0, stdout=b'wazuh-worker2 | CHANGED => {\n    "changed": true,\n    "checksum": "da39a3ee5e6b4b0d3255bfef95601890afd80709",\n    "dest": "/var/ossec/logs/cluster.log",\n    "gid": 999,\n    "group": "wazuh",\n    "md5sum": "d41d8cd98f00b204e9800998ecf8427e",\n    "mode": "0660",\n    "owner": "wazuh",\n    "size": 0,\n    "src": "/root/.ansible/tmp/ansible-tmp-1637924449.572095-182430799415910/source",\n    "state": "file",\n    "uid": 999\n}\n', stderr=None)
INFO     testinfra:ansible.py:71 RUN Ansible('copy', "dest=/var/ossec/logs/cluster.log content='' force=yes", {'check': False, 'become': False}): {'changed': True,
 'checksum': 'da39a3ee5e6b4b0d3255bfef95601890afd80709',
 'dest': '/var/ossec/logs/cluster.log',
 'gid': 999,
 'group': 'wazuh',
 'md5sum': 'd41d8cd98f00b204e9800998ecf8427e',
 'mode': '0660',
 'owner': 'wazuh',
 'size': 0,
 'src': '/root/.ansible/tmp/ansible-tmp-1637924449.572095-182430799415910/source',
 'state': 'file',
 'uid': 999}
DEBUG    testinfra:base.py:286 RUN CommandResult(command=b"ansible --tree /tmp/tmp1pedhm8z -i /home/yanazaeva/git/wazuh-qa/tests/system/provisioning/basic_cluster/inventory.yml -m command --args 'service wazuh-manager restart' wazuh-master", exit_status=0, stdout=b"wazuh-master | CHANGED | rc=0 >>\nKilling wazuh-clusterd...\nKilling wazuh-modulesd...\nKilling wazuh-monitord...\nKilling wazuh-logcollector...\nKilling wazuh-remoted...\nKilling wazuh-syscheckd...\nKilling wazuh-analysisd...\nwazuh-maild not running...\nKilling wazuh-execd...\nKilling wazuh-db...\nKilling wazuh-authd...\nwazuh-agentlessd not running...\nwazuh-integratord not running...\nwazuh-dbd not running...\nwazuh-csyslogd not running...\nKilling wazuh-apid...\nWazuh v4.3.0 Stopped\nStarting Wazuh v4.3.0...\nStarted wazuh-apid...\nStarted wazuh-csyslogd...\nStarted wazuh-dbd...\nStarted wazuh-integratord...\nStarted wazuh-agentlessd...\nStarted wazuh-authd...\nStarted wazuh-db...\nStarted wazuh-execd...\nStarted wazuh-analysisd...\nStarted wazuh-syscheckd...\nStarted wazuh-remoted...\nStarted wazuh-logcollector...\nStarted wazuh-monitord...\nStarted wazuh-modulesd...\nStarted wazuh-clusterd...\nCompleted.2021/11/26 12:00:54 wazuh-db[25164] debug_op.c:70 at _log(): DEBUG: Logging module auto-initialized\n2021/11/26 12:00:54 wazuh-db[25164] main.c:101 at main(): DEBUG: Wazuh home directory: /var/ossec\n2021/11/26 12:00:54 wazuh-modulesd: WARNING: The <ignore_time> tag at module 'vulnerability-detector' is deprecated for version newer than 4.3.\n2021/11/26 12:00:57 wazuh-integratord: INFO: Remote integrations not configured. Clean exit.\n2021/11/26 12:00:57 wazuh-authd[25232] debug_op.c:70 at _log(): DEBUG: Logging module auto-initialized\n2021/11/26 12:00:57 wazuh-authd[25232] config.c:20 at authd_read_config(): DEBUG: Reading configuration 'etc/ossec.conf'\n2021/11/26 12:00:57 wazuh-authd[25232] authd-config.c:109 at Read_Authd(): DEBUG: The <force_insert> tag is deprecated since version 4.3.0.\n2021/11/26 12:00:57 wazuh-authd[25232] authd-config.c:117 at Read_Authd(): DEBUG: The <force_time> tag is deprecated since version 4.3.0.\n2021/11/26 12:00:57 wazuh-authd[25232] authd-config.c:159 at Read_Authd(): DEBUG: The <limit_maxagents> tag is deprecated since version 4.1.0.\n2021/11/26 12:00:57 wazuh-authd[25232] authd-config.c:198 at Read_Authd(): DEBUG: Setting <force><enabled> tag to 'yes' to comply with the legacy <force_insert> option found.\n2021/11/26 12:00:57 wazuh-authd[25232] main-server.c:372 at main(): DEBUG: Wazuh home directory: /var/ossec\n2021/11/26 12:00:58 wazuh-db[25248] debug_op.c:70 at _log(): DEBUG: Logging module auto-initialized\n2021/11/26 12:00:58 wazuh-db[25248] main.c:101 at main(): DEBUG: Wazuh home directory: /var/ossec\n2021/11/26 12:01:01 wazuh-syscheckd: INFO: (6678): No directory provided for syscheck to monitor.\n2021/11/26 12:01:01 wazuh-syscheckd: INFO: (6001): File integrity monitoring disabled.\n2021/11/26 12:01:01 rootcheck: INFO: Rootcheck disabled.\n2021/11/26 12:01:04 wazuh-modulesd: WARNING: The <ignore_time> tag at module 'vulnerability-detector' is deprecated for version newer than 4.3.\n", stderr=b"[WARNING]: Consider using the service module rather than running 'service'.  If\nyou need to use command because service is insufficient you can add 'warn:\nfalse' to this command task or set 'command_warnings=False' in ansible.cfg to\nget rid of this message.\n")
INFO     testinfra:ansible.py:71 RUN Ansible('command', 'service wazuh-manager restart', {'check': False, 'become': False}): {'changed': True,
 'cmd': ['service', 'wazuh-manager', 'restart'],
 'delta': '0:00:15.570062',
 'end': '2021-11-26 12:01:07.445115',
 'rc': 0,
 'start': '2021-11-26 12:00:51.875053',
 'stderr': '2021/11/26 12:00:54 wazuh-db[25164] debug_op.c:70 at _log(): '
           'DEBUG: Logging module auto-initialized\n'
           '2021/11/26 12:00:54 wazuh-db[25164] main.c:101 at main(): DEBUG: '
           'Wazuh home directory: /var/ossec\n'
           '2021/11/26 12:00:54 wazuh-modulesd: WARNING: The <ignore_time> tag '
           "at module 'vulnerability-detector' is deprecated for version newer "
           'than 4.3.\n'
           '2021/11/26 12:00:57 wazuh-integratord: INFO: Remote integrations '
           'not configured. Clean exit.\n'
           '2021/11/26 12:00:57 wazuh-authd[25232] debug_op.c:70 at _log(): '
           'DEBUG: Logging module auto-initialized\n'
           '2021/11/26 12:00:57 wazuh-authd[25232] config.c:20 at '
           'authd_read_config(): DEBUG: Reading configuration '
           "'etc/ossec.conf'\n"
           '2021/11/26 12:00:57 wazuh-authd[25232] authd-config.c:109 at '
           'Read_Authd(): DEBUG: The <force_insert> tag is deprecated since '
           'version 4.3.0.\n'
           '2021/11/26 12:00:57 wazuh-authd[25232] authd-config.c:117 at '
           'Read_Authd(): DEBUG: The <force_time> tag is deprecated since '
           'version 4.3.0.\n'
           '2021/11/26 12:00:57 wazuh-authd[25232] authd-config.c:159 at '
           'Read_Authd(): DEBUG: The <limit_maxagents> tag is deprecated since '
           'version 4.1.0.\n'
           '2021/11/26 12:00:57 wazuh-authd[25232] authd-config.c:198 at '
           "Read_Authd(): DEBUG: Setting <force><enabled> tag to 'yes' to "
           'comply with the legacy <force_insert> option found.\n'
           '2021/11/26 12:00:57 wazuh-authd[25232] main-server.c:372 at '
           'main(): DEBUG: Wazuh home directory: /var/ossec\n'
           '2021/11/26 12:00:58 wazuh-db[25248] debug_op.c:70 at _log(): '
           'DEBUG: Logging module auto-initialized\n'
           '2021/11/26 12:00:58 wazuh-db[25248] main.c:101 at main(): DEBUG: '
           'Wazuh home directory: /var/ossec\n'
           '2021/11/26 12:01:01 wazuh-syscheckd: INFO: (6678): No directory '
           'provided for syscheck to monitor.\n'
           '2021/11/26 12:01:01 wazuh-syscheckd: INFO: (6001): File integrity '
           'monitoring disabled.\n'
           '2021/11/26 12:01:01 rootcheck: INFO: Rootcheck disabled.\n'
           '2021/11/26 12:01:04 wazuh-modulesd: WARNING: The <ignore_time> tag '
           "at module 'vulnerability-detector' is deprecated for version newer "
           'than 4.3.',
 'stderr_lines': ['2021/11/26 12:00:54 wazuh-db[25164] debug_op.c:70 at '
                  '_log(): DEBUG: Logging module auto-initialized',
                  '2021/11/26 12:00:54 wazuh-db[25164] main.c:101 at main(): '
                  'DEBUG: Wazuh home directory: /var/ossec',
                  '2021/11/26 12:00:54 wazuh-modulesd: WARNING: The '
                  "<ignore_time> tag at module 'vulnerability-detector' is "
                  'deprecated for version newer than 4.3.',
                  '2021/11/26 12:00:57 wazuh-integratord: INFO: Remote '
                  'integrations not configured. Clean exit.',
                  '2021/11/26 12:00:57 wazuh-authd[25232] debug_op.c:70 at '
                  '_log(): DEBUG: Logging module auto-initialized',
                  '2021/11/26 12:00:57 wazuh-authd[25232] config.c:20 at '
                  'authd_read_config(): DEBUG: Reading configuration '
                  "'etc/ossec.conf'",
                  '2021/11/26 12:00:57 wazuh-authd[25232] authd-config.c:109 '
                  'at Read_Authd(): DEBUG: The <force_insert> tag is '
                  'deprecated since version 4.3.0.',
                  '2021/11/26 12:00:57 wazuh-authd[25232] authd-config.c:117 '
                  'at Read_Authd(): DEBUG: The <force_time> tag is deprecated '
                  'since version 4.3.0.',
                  '2021/11/26 12:00:57 wazuh-authd[25232] authd-config.c:159 '
                  'at Read_Authd(): DEBUG: The <limit_maxagents> tag is '
                  'deprecated since version 4.1.0.',
                  '2021/11/26 12:00:57 wazuh-authd[25232] authd-config.c:198 '
                  'at Read_Authd(): DEBUG: Setting <force><enabled> tag to '
                  "'yes' to comply with the legacy <force_insert> option "
                  'found.',
                  '2021/11/26 12:00:57 wazuh-authd[25232] main-server.c:372 at '
                  'main(): DEBUG: Wazuh home directory: /var/ossec',
                  '2021/11/26 12:00:58 wazuh-db[25248] debug_op.c:70 at '
                  '_log(): DEBUG: Logging module auto-initialized',
                  '2021/11/26 12:00:58 wazuh-db[25248] main.c:101 at main(): '
                  'DEBUG: Wazuh home directory: /var/ossec',
                  '2021/11/26 12:01:01 wazuh-syscheckd: INFO: (6678): No '
                  'directory provided for syscheck to monitor.',
                  '2021/11/26 12:01:01 wazuh-syscheckd: INFO: (6001): File '
                  'integrity monitoring disabled.',
                  '2021/11/26 12:01:01 rootcheck: INFO: Rootcheck disabled.',
                  '2021/11/26 12:01:04 wazuh-modulesd: WARNING: The '
                  "<ignore_time> tag at module 'vulnerability-detector' is "
                  'deprecated for version newer than 4.3.'],
 'stdout': 'Killing wazuh-clusterd...\n'
           'Killing wazuh-modulesd...\n'
           'Killing wazuh-monitord...\n'
           'Killing wazuh-logcollector...\n'
           'Killing wazuh-remoted...\n'
           'Killing wazuh-syscheckd...\n'
           'Killing wazuh-analysisd...\n'
           'wazuh-maild not running...\n'
           'Killing wazuh-execd...\n'
           'Killing wazuh-db...\n'
           'Killing wazuh-authd...\n'
           'wazuh-agentlessd not running...\n'
           'wazuh-integratord not running...\n'
           'wazuh-dbd not running...\n'
           'wazuh-csyslogd not running...\n'
           'Killing wazuh-apid...\n'
           'Wazuh v4.3.0 Stopped\n'
           'Starting Wazuh v4.3.0...\n'
           'Started wazuh-apid...\n'
           'Started wazuh-csyslogd...\n'
           'Started wazuh-dbd...\n'
           'Started wazuh-integratord...\n'
           'Started wazuh-agentlessd...\n'
           'Started wazuh-authd...\n'
           'Started wazuh-db...\n'
           'Started wazuh-execd...\n'
           'Started wazuh-analysisd...\n'
           'Started wazuh-syscheckd...\n'
           'Started wazuh-remoted...\n'
           'Started wazuh-logcollector...\n'
           'Started wazuh-monitord...\n'
           'Started wazuh-modulesd...\n'
           'Started wazuh-clusterd...\n'
           'Completed.',
 'stdout_lines': ['Killing wazuh-clusterd...',
                  'Killing wazuh-modulesd...',
                  'Killing wazuh-monitord...',
                  'Killing wazuh-logcollector...',
                  'Killing wazuh-remoted...',
                  'Killing wazuh-syscheckd...',
                  'Killing wazuh-analysisd...',
                  'wazuh-maild not running...',
                  'Killing wazuh-execd...',
                  'Killing wazuh-db...',
                  'Killing wazuh-authd...',
                  'wazuh-agentlessd not running...',
                  'wazuh-integratord not running...',
                  'wazuh-dbd not running...',
                  'wazuh-csyslogd not running...',
                  'Killing wazuh-apid...',
                  'Wazuh v4.3.0 Stopped',
                  'Starting Wazuh v4.3.0...',
                  'Started wazuh-apid...',
                  'Started wazuh-csyslogd...',
                  'Started wazuh-dbd...',
                  'Started wazuh-integratord...',
                  'Started wazuh-agentlessd...',
                  'Started wazuh-authd...',
                  'Started wazuh-db...',
                  'Started wazuh-execd...',
                  'Started wazuh-analysisd...',
                  'Started wazuh-syscheckd...',
                  'Started wazuh-remoted...',
                  'Started wazuh-logcollector...',
                  'Started wazuh-monitord...',
                  'Started wazuh-modulesd...',
                  'Started wazuh-clusterd...',
                  'Completed.'],
 'warnings': ['Consider using the service module rather than running '
              "'service'.  If you need to use command because service is "
              "insufficient you can add 'warn: false' to this command task or "
              "set 'command_warnings=False' in ansible.cfg to get rid of this "
              'message.']}
DEBUG    testinfra:base.py:286 RUN CommandResult(command=b"ansible --tree /tmp/tmpe5npa5x9 -i /home/yanazaeva/git/wazuh-qa/tests/system/provisioning/basic_cluster/inventory.yml -m command --args 'service wazuh-manager restart' wazuh-worker1", exit_status=0, stdout=b"wazuh-worker1 | CHANGED | rc=0 >>\nKilling wazuh-clusterd...\nKilling wazuh-modulesd...\nKilling wazuh-monitord...\nKilling wazuh-logcollector...\nKilling wazuh-remoted...\nKilling wazuh-syscheckd...\nKilling wazuh-analysisd...\nwazuh-maild not running...\nKilling wazuh-execd...\nKilling wazuh-db...\nKilling wazuh-authd...\nwazuh-agentlessd not running...\nwazuh-integratord not running...\nwazuh-dbd not running...\nwazuh-csyslogd not running...\nKilling wazuh-apid...\nWazuh v4.3.0 Stopped\nStarting Wazuh v4.3.0...\nStarted wazuh-apid...\nStarted wazuh-csyslogd...\nStarted wazuh-dbd...\nStarted wazuh-integratord...\nStarted wazuh-agentlessd...\nStarted wazuh-authd...\nStarted wazuh-db...\nStarted wazuh-execd...\nStarted wazuh-analysisd...\nStarted wazuh-syscheckd...\nStarted wazuh-remoted...\nStarted wazuh-logcollector...\nStarted wazuh-monitord...\nStarted wazuh-modulesd...\nStarted wazuh-clusterd...\nCompleted.2021/11/26 12:01:11 wazuh-db[20385] debug_op.c:70 at _log(): DEBUG: Logging module auto-initialized\n2021/11/26 12:01:11 wazuh-db[20385] main.c:101 at main(): DEBUG: Wazuh home directory: /var/ossec\n2021/11/26 12:01:11 wazuh-modulesd: WARNING: The <ignore_time> tag at module 'vulnerability-detector' is deprecated for version newer than 4.3.\n2021/11/26 12:01:14 wazuh-integratord: INFO: Remote integrations not configured. Clean exit.\n2021/11/26 12:01:14 wazuh-authd[20453] debug_op.c:70 at _log(): DEBUG: Logging module auto-initialized\n2021/11/26 12:01:14 wazuh-authd[20453] config.c:20 at authd_read_config(): DEBUG: Reading configuration 'etc/ossec.conf'\n2021/11/26 12:01:14 wazuh-authd[20453] authd-config.c:109 at Read_Authd(): DEBUG: The <force_insert> tag is deprecated since version 4.3.0.\n2021/11/26 12:01:14 wazuh-authd[20453] authd-config.c:117 at Read_Authd(): DEBUG: The <force_time> tag is deprecated since version 4.3.0.\n2021/11/26 12:01:14 wazuh-authd[20453] authd-config.c:159 at Read_Authd(): DEBUG: The <limit_maxagents> tag is deprecated since version 4.1.0.\n2021/11/26 12:01:14 wazuh-authd[20453] authd-config.c:198 at Read_Authd(): DEBUG: Setting <force><enabled> tag to 'yes' to comply with the legacy <force_insert> option found.\n2021/11/26 12:01:14 wazuh-authd[20453] main-server.c:372 at main(): DEBUG: Wazuh home directory: /var/ossec\n2021/11/26 12:01:15 wazuh-db[20468] debug_op.c:70 at _log(): DEBUG: Logging module auto-initialized\n2021/11/26 12:01:15 wazuh-db[20468] main.c:101 at main(): DEBUG: Wazuh home directory: /var/ossec\n2021/11/26 12:01:18 wazuh-syscheckd: INFO: (6678): No directory provided for syscheck to monitor.\n2021/11/26 12:01:18 wazuh-syscheckd: INFO: (6001): File integrity monitoring disabled.\n2021/11/26 12:01:18 rootcheck: INFO: Rootcheck disabled.\n2021/11/26 12:01:22 wazuh-modulesd: WARNING: The <ignore_time> tag at module 'vulnerability-detector' is deprecated for version newer than 4.3.\n", stderr=b"[WARNING]: Consider using the service module rather than running 'service'.  If\nyou need to use command because service is insufficient you can add 'warn:\nfalse' to this command task or set 'command_warnings=False' in ansible.cfg to\nget rid of this message.\n")
INFO     testinfra:ansible.py:71 RUN Ansible('command', 'service wazuh-manager restart', {'check': False, 'become': False}): {'changed': True,
 'cmd': ['service', 'wazuh-manager', 'restart'],
 'delta': '0:00:16.889824',
 'end': '2021-11-26 12:01:25.393679',
 'rc': 0,
 'start': '2021-11-26 12:01:08.503855',
 'stderr': '2021/11/26 12:01:11 wazuh-db[20385] debug_op.c:70 at _log(): '
           'DEBUG: Logging module auto-initialized\n'
           '2021/11/26 12:01:11 wazuh-db[20385] main.c:101 at main(): DEBUG: '
           'Wazuh home directory: /var/ossec\n'
           '2021/11/26 12:01:11 wazuh-modulesd: WARNING: The <ignore_time> tag '
           "at module 'vulnerability-detector' is deprecated for version newer "
           'than 4.3.\n'
           '2021/11/26 12:01:14 wazuh-integratord: INFO: Remote integrations '
           'not configured. Clean exit.\n'
           '2021/11/26 12:01:14 wazuh-authd[20453] debug_op.c:70 at _log(): '
           'DEBUG: Logging module auto-initialized\n'
           '2021/11/26 12:01:14 wazuh-authd[20453] config.c:20 at '
           'authd_read_config(): DEBUG: Reading configuration '
           "'etc/ossec.conf'\n"
           '2021/11/26 12:01:14 wazuh-authd[20453] authd-config.c:109 at '
           'Read_Authd(): DEBUG: The <force_insert> tag is deprecated since '
           'version 4.3.0.\n'
           '2021/11/26 12:01:14 wazuh-authd[20453] authd-config.c:117 at '
           'Read_Authd(): DEBUG: The <force_time> tag is deprecated since '
           'version 4.3.0.\n'
           '2021/11/26 12:01:14 wazuh-authd[20453] authd-config.c:159 at '
           'Read_Authd(): DEBUG: The <limit_maxagents> tag is deprecated since '
           'version 4.1.0.\n'
           '2021/11/26 12:01:14 wazuh-authd[20453] authd-config.c:198 at '
           "Read_Authd(): DEBUG: Setting <force><enabled> tag to 'yes' to "
           'comply with the legacy <force_insert> option found.\n'
           '2021/11/26 12:01:14 wazuh-authd[20453] main-server.c:372 at '
           'main(): DEBUG: Wazuh home directory: /var/ossec\n'
           '2021/11/26 12:01:15 wazuh-db[20468] debug_op.c:70 at _log(): '
           'DEBUG: Logging module auto-initialized\n'
           '2021/11/26 12:01:15 wazuh-db[20468] main.c:101 at main(): DEBUG: '
           'Wazuh home directory: /var/ossec\n'
           '2021/11/26 12:01:18 wazuh-syscheckd: INFO: (6678): No directory '
           'provided for syscheck to monitor.\n'
           '2021/11/26 12:01:18 wazuh-syscheckd: INFO: (6001): File integrity '
           'monitoring disabled.\n'
           '2021/11/26 12:01:18 rootcheck: INFO: Rootcheck disabled.\n'
           '2021/11/26 12:01:22 wazuh-modulesd: WARNING: The <ignore_time> tag '
           "at module 'vulnerability-detector' is deprecated for version newer "
           'than 4.3.',
 'stderr_lines': ['2021/11/26 12:01:11 wazuh-db[20385] debug_op.c:70 at '
                  '_log(): DEBUG: Logging module auto-initialized',
                  '2021/11/26 12:01:11 wazuh-db[20385] main.c:101 at main(): '
                  'DEBUG: Wazuh home directory: /var/ossec',
                  '2021/11/26 12:01:11 wazuh-modulesd: WARNING: The '
                  "<ignore_time> tag at module 'vulnerability-detector' is "
                  'deprecated for version newer than 4.3.',
                  '2021/11/26 12:01:14 wazuh-integratord: INFO: Remote '
                  'integrations not configured. Clean exit.',
                  '2021/11/26 12:01:14 wazuh-authd[20453] debug_op.c:70 at '
                  '_log(): DEBUG: Logging module auto-initialized',
                  '2021/11/26 12:01:14 wazuh-authd[20453] config.c:20 at '
                  'authd_read_config(): DEBUG: Reading configuration '
                  "'etc/ossec.conf'",
                  '2021/11/26 12:01:14 wazuh-authd[20453] authd-config.c:109 '
                  'at Read_Authd(): DEBUG: The <force_insert> tag is '
                  'deprecated since version 4.3.0.',
                  '2021/11/26 12:01:14 wazuh-authd[20453] authd-config.c:117 '
                  'at Read_Authd(): DEBUG: The <force_time> tag is deprecated '
                  'since version 4.3.0.',
                  '2021/11/26 12:01:14 wazuh-authd[20453] authd-config.c:159 '
                  'at Read_Authd(): DEBUG: The <limit_maxagents> tag is '
                  'deprecated since version 4.1.0.',
                  '2021/11/26 12:01:14 wazuh-authd[20453] authd-config.c:198 '
                  'at Read_Authd(): DEBUG: Setting <force><enabled> tag to '
                  "'yes' to comply with the legacy <force_insert> option "
                  'found.',
                  '2021/11/26 12:01:14 wazuh-authd[20453] main-server.c:372 at '
                  'main(): DEBUG: Wazuh home directory: /var/ossec',
                  '2021/11/26 12:01:15 wazuh-db[20468] debug_op.c:70 at '
                  '_log(): DEBUG: Logging module auto-initialized',
                  '2021/11/26 12:01:15 wazuh-db[20468] main.c:101 at main(): '
                  'DEBUG: Wazuh home directory: /var/ossec',
                  '2021/11/26 12:01:18 wazuh-syscheckd: INFO: (6678): No '
                  'directory provided for syscheck to monitor.',
                  '2021/11/26 12:01:18 wazuh-syscheckd: INFO: (6001): File '
                  'integrity monitoring disabled.',
                  '2021/11/26 12:01:18 rootcheck: INFO: Rootcheck disabled.',
                  '2021/11/26 12:01:22 wazuh-modulesd: WARNING: The '
                  "<ignore_time> tag at module 'vulnerability-detector' is "
                  'deprecated for version newer than 4.3.'],
 'stdout': 'Killing wazuh-clusterd...\n'
           'Killing wazuh-modulesd...\n'
           'Killing wazuh-monitord...\n'
           'Killing wazuh-logcollector...\n'
           'Killing wazuh-remoted...\n'
           'Killing wazuh-syscheckd...\n'
           'Killing wazuh-analysisd...\n'
           'wazuh-maild not running...\n'
           'Killing wazuh-execd...\n'
           'Killing wazuh-db...\n'
           'Killing wazuh-authd...\n'
           'wazuh-agentlessd not running...\n'
           'wazuh-integratord not running...\n'
           'wazuh-dbd not running...\n'
           'wazuh-csyslogd not running...\n'
           'Killing wazuh-apid...\n'
           'Wazuh v4.3.0 Stopped\n'
           'Starting Wazuh v4.3.0...\n'
           'Started wazuh-apid...\n'
           'Started wazuh-csyslogd...\n'
           'Started wazuh-dbd...\n'
           'Started wazuh-integratord...\n'
           'Started wazuh-agentlessd...\n'
           'Started wazuh-authd...\n'
           'Started wazuh-db...\n'
           'Started wazuh-execd...\n'
           'Started wazuh-analysisd...\n'
           'Started wazuh-syscheckd...\n'
           'Started wazuh-remoted...\n'
           'Started wazuh-logcollector...\n'
           'Started wazuh-monitord...\n'
           'Started wazuh-modulesd...\n'
           'Started wazuh-clusterd...\n'
           'Completed.',
 'stdout_lines': ['Killing wazuh-clusterd...',
                  'Killing wazuh-modulesd...',
                  'Killing wazuh-monitord...',
                  'Killing wazuh-logcollector...',
                  'Killing wazuh-remoted...',
                  'Killing wazuh-syscheckd...',
                  'Killing wazuh-analysisd...',
                  'wazuh-maild not running...',
                  'Killing wazuh-execd...',
                  'Killing wazuh-db...',
                  'Killing wazuh-authd...',
                  'wazuh-agentlessd not running...',
                  'wazuh-integratord not running...',
                  'wazuh-dbd not running...',
                  'wazuh-csyslogd not running...',
                  'Killing wazuh-apid...',
                  'Wazuh v4.3.0 Stopped',
                  'Starting Wazuh v4.3.0...',
                  'Started wazuh-apid...',
                  'Started wazuh-csyslogd...',
                  'Started wazuh-dbd...',
                  'Started wazuh-integratord...',
                  'Started wazuh-agentlessd...',
                  'Started wazuh-authd...',
                  'Started wazuh-db...',
                  'Started wazuh-execd...',
                  'Started wazuh-analysisd...',
                  'Started wazuh-syscheckd...',
                  'Started wazuh-remoted...',
                  'Started wazuh-logcollector...',
                  'Started wazuh-monitord...',
                  'Started wazuh-modulesd...',
                  'Started wazuh-clusterd...',
                  'Completed.'],
 'warnings': ['Consider using the service module rather than running '
              "'service'.  If you need to use command because service is "
              "insufficient you can add 'warn: false' to this command task or "
              "set 'command_warnings=False' in ansible.cfg to get rid of this "
              'message.']}
DEBUG    testinfra:base.py:286 RUN CommandResult(command=b"ansible --tree /tmp/tmprepbqu3g -i /home/yanazaeva/git/wazuh-qa/tests/system/provisioning/basic_cluster/inventory.yml -m command --args 'service wazuh-manager restart' wazuh-worker2", exit_status=0, stdout=b"wazuh-worker2 | CHANGED | rc=0 >>\nKilling wazuh-clusterd...\nKilling wazuh-modulesd...\nKilling wazuh-monitord...\nKilling wazuh-logcollector...\nKilling wazuh-remoted...\nKilling wazuh-syscheckd...\nKilling wazuh-analysisd...\nwazuh-maild not running...\nKilling wazuh-execd...\nKilling wazuh-db...\nKilling wazuh-authd...\nwazuh-agentlessd not running...\nwazuh-integratord not running...\nwazuh-dbd not running...\nwazuh-csyslogd not running...\nKilling wazuh-apid...\nWazuh v4.3.0 Stopped\nStarting Wazuh v4.3.0...\nStarted wazuh-apid...\nStarted wazuh-csyslogd...\nStarted wazuh-dbd...\nStarted wazuh-integratord...\nStarted wazuh-agentlessd...\nStarted wazuh-authd...\nStarted wazuh-db...\nStarted wazuh-execd...\nStarted wazuh-analysisd...\nStarted wazuh-syscheckd...\nStarted wazuh-remoted...\nStarted wazuh-logcollector...\nStarted wazuh-monitord...\nStarted wazuh-modulesd...\nStarted wazuh-clusterd...\nCompleted.2021/11/26 12:01:29 wazuh-db[20589] debug_op.c:70 at _log(): DEBUG: Logging module auto-initialized\n2021/11/26 12:01:29 wazuh-db[20589] main.c:101 at main(): DEBUG: Wazuh home directory: /var/ossec\n2021/11/26 12:01:29 wazuh-modulesd: WARNING: The <ignore_time> tag at module 'vulnerability-detector' is deprecated for version newer than 4.3.\n2021/11/26 12:01:32 wazuh-integratord: INFO: Remote integrations not configured. Clean exit.\n2021/11/26 12:01:32 wazuh-authd[20657] debug_op.c:70 at _log(): DEBUG: Logging module auto-initialized\n2021/11/26 12:01:32 wazuh-authd[20657] config.c:20 at authd_read_config(): DEBUG: Reading configuration 'etc/ossec.conf'\n2021/11/26 12:01:32 wazuh-authd[20657] authd-config.c:109 at Read_Authd(): DEBUG: The <force_insert> tag is deprecated since version 4.3.0.\n2021/11/26 12:01:32 wazuh-authd[20657] authd-config.c:117 at Read_Authd(): DEBUG: The <force_time> tag is deprecated since version 4.3.0.\n2021/11/26 12:01:32 wazuh-authd[20657] authd-config.c:159 at Read_Authd(): DEBUG: The <limit_maxagents> tag is deprecated since version 4.1.0.\n2021/11/26 12:01:32 wazuh-authd[20657] authd-config.c:198 at Read_Authd(): DEBUG: Setting <force><enabled> tag to 'yes' to comply with the legacy <force_insert> option found.\n2021/11/26 12:01:32 wazuh-authd[20657] main-server.c:372 at main(): DEBUG: Wazuh home directory: /var/ossec\n2021/11/26 12:01:33 wazuh-db[20672] debug_op.c:70 at _log(): DEBUG: Logging module auto-initialized\n2021/11/26 12:01:33 wazuh-db[20672] main.c:101 at main(): DEBUG: Wazuh home directory: /var/ossec\n2021/11/26 12:01:35 wazuh-syscheckd: INFO: (6678): No directory provided for syscheck to monitor.\n2021/11/26 12:01:35 wazuh-syscheckd: INFO: (6001): File integrity monitoring disabled.\n2021/11/26 12:01:35 rootcheck: INFO: Rootcheck disabled.\n2021/11/26 12:01:38 wazuh-modulesd: WARNING: The <ignore_time> tag at module 'vulnerability-detector' is deprecated for version newer than 4.3.\n", stderr=b"[WARNING]: Consider using the service module rather than running 'service'.  If\nyou need to use command because service is insufficient you can add 'warn:\nfalse' to this command task or set 'command_warnings=False' in ansible.cfg to\nget rid of this message.\n")
INFO     testinfra:ansible.py:71 RUN Ansible('command', 'service wazuh-manager restart', {'check': False, 'become': False}): {'changed': True,
 'cmd': ['service', 'wazuh-manager', 'restart'],
 'delta': '0:00:14.943368',
 'end': '2021-11-26 12:01:41.342545',
 'rc': 0,
 'start': '2021-11-26 12:01:26.399177',
 'stderr': '2021/11/26 12:01:29 wazuh-db[20589] debug_op.c:70 at _log(): '
           'DEBUG: Logging module auto-initialized\n'
           '2021/11/26 12:01:29 wazuh-db[20589] main.c:101 at main(): DEBUG: '
           'Wazuh home directory: /var/ossec\n'
           '2021/11/26 12:01:29 wazuh-modulesd: WARNING: The <ignore_time> tag '
           "at module 'vulnerability-detector' is deprecated for version newer "
           'than 4.3.\n'
           '2021/11/26 12:01:32 wazuh-integratord: INFO: Remote integrations '
           'not configured. Clean exit.\n'
           '2021/11/26 12:01:32 wazuh-authd[20657] debug_op.c:70 at _log(): '
           'DEBUG: Logging module auto-initialized\n'
           '2021/11/26 12:01:32 wazuh-authd[20657] config.c:20 at '
           'authd_read_config(): DEBUG: Reading configuration '
           "'etc/ossec.conf'\n"
           '2021/11/26 12:01:32 wazuh-authd[20657] authd-config.c:109 at '
           'Read_Authd(): DEBUG: The <force_insert> tag is deprecated since '
           'version 4.3.0.\n'
           '2021/11/26 12:01:32 wazuh-authd[20657] authd-config.c:117 at '
           'Read_Authd(): DEBUG: The <force_time> tag is deprecated since '
           'version 4.3.0.\n'
           '2021/11/26 12:01:32 wazuh-authd[20657] authd-config.c:159 at '
           'Read_Authd(): DEBUG: The <limit_maxagents> tag is deprecated since '
           'version 4.1.0.\n'
           '2021/11/26 12:01:32 wazuh-authd[20657] authd-config.c:198 at '
           "Read_Authd(): DEBUG: Setting <force><enabled> tag to 'yes' to "
           'comply with the legacy <force_insert> option found.\n'
           '2021/11/26 12:01:32 wazuh-authd[20657] main-server.c:372 at '
           'main(): DEBUG: Wazuh home directory: /var/ossec\n'
           '2021/11/26 12:01:33 wazuh-db[20672] debug_op.c:70 at _log(): '
           'DEBUG: Logging module auto-initialized\n'
           '2021/11/26 12:01:33 wazuh-db[20672] main.c:101 at main(): DEBUG: '
           'Wazuh home directory: /var/ossec\n'
           '2021/11/26 12:01:35 wazuh-syscheckd: INFO: (6678): No directory '
           'provided for syscheck to monitor.\n'
           '2021/11/26 12:01:35 wazuh-syscheckd: INFO: (6001): File integrity '
           'monitoring disabled.\n'
           '2021/11/26 12:01:35 rootcheck: INFO: Rootcheck disabled.\n'
           '2021/11/26 12:01:38 wazuh-modulesd: WARNING: The <ignore_time> tag '
           "at module 'vulnerability-detector' is deprecated for version newer "
           'than 4.3.',
 'stderr_lines': ['2021/11/26 12:01:29 wazuh-db[20589] debug_op.c:70 at '
                  '_log(): DEBUG: Logging module auto-initialized',
                  '2021/11/26 12:01:29 wazuh-db[20589] main.c:101 at main(): '
                  'DEBUG: Wazuh home directory: /var/ossec',
                  '2021/11/26 12:01:29 wazuh-modulesd: WARNING: The '
                  "<ignore_time> tag at module 'vulnerability-detector' is "
                  'deprecated for version newer than 4.3.',
                  '2021/11/26 12:01:32 wazuh-integratord: INFO: Remote '
                  'integrations not configured. Clean exit.',
                  '2021/11/26 12:01:32 wazuh-authd[20657] debug_op.c:70 at '
                  '_log(): DEBUG: Logging module auto-initialized',
                  '2021/11/26 12:01:32 wazuh-authd[20657] config.c:20 at '
                  'authd_read_config(): DEBUG: Reading configuration '
                  "'etc/ossec.conf'",
                  '2021/11/26 12:01:32 wazuh-authd[20657] authd-config.c:109 '
                  'at Read_Authd(): DEBUG: The <force_insert> tag is '
                  'deprecated since version 4.3.0.',
                  '2021/11/26 12:01:32 wazuh-authd[20657] authd-config.c:117 '
                  'at Read_Authd(): DEBUG: The <force_time> tag is deprecated '
                  'since version 4.3.0.',
                  '2021/11/26 12:01:32 wazuh-authd[20657] authd-config.c:159 '
                  'at Read_Authd(): DEBUG: The <limit_maxagents> tag is '
                  'deprecated since version 4.1.0.',
                  '2021/11/26 12:01:32 wazuh-authd[20657] authd-config.c:198 '
                  'at Read_Authd(): DEBUG: Setting <force><enabled> tag to '
                  "'yes' to comply with the legacy <force_insert> option "
                  'found.',
                  '2021/11/26 12:01:32 wazuh-authd[20657] main-server.c:372 at '
                  'main(): DEBUG: Wazuh home directory: /var/ossec',
                  '2021/11/26 12:01:33 wazuh-db[20672] debug_op.c:70 at '
                  '_log(): DEBUG: Logging module auto-initialized',
                  '2021/11/26 12:01:33 wazuh-db[20672] main.c:101 at main(): '
                  'DEBUG: Wazuh home directory: /var/ossec',
                  '2021/11/26 12:01:35 wazuh-syscheckd: INFO: (6678): No '
                  'directory provided for syscheck to monitor.',
                  '2021/11/26 12:01:35 wazuh-syscheckd: INFO: (6001): File '
                  'integrity monitoring disabled.',
                  '2021/11/26 12:01:35 rootcheck: INFO: Rootcheck disabled.',
                  '2021/11/26 12:01:38 wazuh-modulesd: WARNING: The '
                  "<ignore_time> tag at module 'vulnerability-detector' is "
                  'deprecated for version newer than 4.3.'],
 'stdout': 'Killing wazuh-clusterd...\n'
           'Killing wazuh-modulesd...\n'
           'Killing wazuh-monitord...\n'
           'Killing wazuh-logcollector...\n'
           'Killing wazuh-remoted...\n'
           'Killing wazuh-syscheckd...\n'
           'Killing wazuh-analysisd...\n'
           'wazuh-maild not running...\n'
           'Killing wazuh-execd...\n'
           'Killing wazuh-db...\n'
           'Killing wazuh-authd...\n'
           'wazuh-agentlessd not running...\n'
           'wazuh-integratord not running...\n'
           'wazuh-dbd not running...\n'
           'wazuh-csyslogd not running...\n'
           'Killing wazuh-apid...\n'
           'Wazuh v4.3.0 Stopped\n'
           'Starting Wazuh v4.3.0...\n'
           'Started wazuh-apid...\n'
           'Started wazuh-csyslogd...\n'
           'Started wazuh-dbd...\n'
           'Started wazuh-integratord...\n'
           'Started wazuh-agentlessd...\n'
           'Started wazuh-authd...\n'
           'Started wazuh-db...\n'
           'Started wazuh-execd...\n'
           'Started wazuh-analysisd...\n'
           'Started wazuh-syscheckd...\n'
           'Started wazuh-remoted...\n'
           'Started wazuh-logcollector...\n'
           'Started wazuh-monitord...\n'
           'Started wazuh-modulesd...\n'
           'Started wazuh-clusterd...\n'
           'Completed.',
 'stdout_lines': ['Killing wazuh-clusterd...',
                  'Killing wazuh-modulesd...',
                  'Killing wazuh-monitord...',
                  'Killing wazuh-logcollector...',
                  'Killing wazuh-remoted...',
                  'Killing wazuh-syscheckd...',
                  'Killing wazuh-analysisd...',
                  'wazuh-maild not running...',
                  'Killing wazuh-execd...',
                  'Killing wazuh-db...',
                  'Killing wazuh-authd...',
                  'wazuh-agentlessd not running...',
                  'wazuh-integratord not running...',
                  'wazuh-dbd not running...',
                  'wazuh-csyslogd not running...',
                  'Killing wazuh-apid...',
                  'Wazuh v4.3.0 Stopped',
                  'Starting Wazuh v4.3.0...',
                  'Started wazuh-apid...',
                  'Started wazuh-csyslogd...',
                  'Started wazuh-dbd...',
                  'Started wazuh-integratord...',
                  'Started wazuh-agentlessd...',
                  'Started wazuh-authd...',
                  'Started wazuh-db...',
                  'Started wazuh-execd...',
                  'Started wazuh-analysisd...',
                  'Started wazuh-syscheckd...',
                  'Started wazuh-remoted...',
                  'Started wazuh-logcollector...',
                  'Started wazuh-monitord...',
                  'Started wazuh-modulesd...',
                  'Started wazuh-clusterd...',
                  'Completed.'],
 'warnings': ['Consider using the service module rather than running '
              "'service'.  If you need to use command because service is "
              "insufficient you can add 'warn: false' to this command task or "
              "set 'command_warnings=False' in ansible.cfg to get rid of this "
              'message.']}
------------------------------------------------------------------------------------------- Captured log call -------------------------------------------------------------------------------------------
DEBUG    testinfra:base.py:286 RUN CommandResult(command=b'ansible --tree /tmp/tmpkbosvdv2 -i /home/yanazaeva/git/wazuh-qa/tests/system/provisioning/basic_cluster/inventory.yml -m replace --args \'path=/var/ossec/etc/ossec.conf regexp=\'"\'"\'</client>[\\s\\S]+<client_buffer>\'"\'"\' replace=\'"\'"\'</client><labels><label key="test_label">value</label></labels><client_buffer>\'"\'"\'\' wazuh-agent2', exit_status=0, stdout=b'wazuh-agent2 | CHANGED => {\n    "changed": true,\n    "msg": "1 replacements made"\n}\n', stderr=None)
INFO     testinfra:ansible.py:71 RUN Ansible('replace', 'path=/var/ossec/etc/ossec.conf regexp=\'</client>[\\s\\S]+<client_buffer>\' replace=\'</client><labels><label key="test_label">value</label></labels><client_buffer>\'', {'check': False, 'become': False}): {'changed': True, 'msg': '1 replacements made'}
DEBUG    testinfra:base.py:286 RUN CommandResult(command=b"ansible --tree /tmp/tmpeeljl47o -i /home/yanazaeva/git/wazuh-qa/tests/system/provisioning/basic_cluster/inventory.yml -m command --args 'service wazuh-agent restart' wazuh-agent2", exit_status=0, stdout=b'wazuh-agent2 | CHANGED | rc=0 >>\nKilling wazuh-modulesd... \nKilling wazuh-logcollector... \nKilling wazuh-syscheckd... \nKilling wazuh-agentd... \nKilling wazuh-execd... \nWazuh v4.3.0 Stopped\nStarting Wazuh v4.3.0...\nStarted wazuh-execd...\nStarted wazuh-agentd...\nStarted wazuh-syscheckd...\nStarted wazuh-logcollector...\nStarted wazuh-modulesd...\nCompleted.\n', stderr=b"[WARNING]: Consider using the service module rather than running 'service'.  If\nyou need to use command because service is insufficient you can add 'warn:\nfalse' to this command task or set 'command_warnings=False' in ansible.cfg to\nget rid of this message.\n")
INFO     testinfra:ansible.py:71 RUN Ansible('command', 'service wazuh-agent restart', {'check': False, 'become': False}): {'changed': True,
 'cmd': ['service', 'wazuh-agent', 'restart'],
 'delta': '0:00:07.657728',
 'end': '2021-11-26 12:01:51.223019',
 'rc': 0,
 'start': '2021-11-26 12:01:43.565291',
 'stderr': '',
 'stderr_lines': [],
 'stdout': 'Killing wazuh-modulesd... \n'
           'Killing wazuh-logcollector... \n'
           'Killing wazuh-syscheckd... \n'
           'Killing wazuh-agentd... \n'
           'Killing wazuh-execd... \n'
           'Wazuh v4.3.0 Stopped\n'
           'Starting Wazuh v4.3.0...\n'
           'Started wazuh-execd...\n'
           'Started wazuh-agentd...\n'
           'Started wazuh-syscheckd...\n'
           'Started wazuh-logcollector...\n'
           'Started wazuh-modulesd...\n'
           'Completed.',
 'stdout_lines': ['Killing wazuh-modulesd... ',
                  'Killing wazuh-logcollector... ',
                  'Killing wazuh-syscheckd... ',
                  'Killing wazuh-agentd... ',
                  'Killing wazuh-execd... ',
                  'Wazuh v4.3.0 Stopped',
                  'Starting Wazuh v4.3.0...',
                  'Started wazuh-execd...',
                  'Started wazuh-agentd...',
                  'Started wazuh-syscheckd...',
                  'Started wazuh-logcollector...',
                  'Started wazuh-modulesd...',
                  'Completed.'],
 'warnings': ['Consider using the service module rather than running '
              "'service'.  If you need to use command because service is "
              "insufficient you can add 'warn: false' to this command task or "
              "set 'command_warnings=False' in ansible.cfg to get rid of this "
              'message.']}
DEBUG    wazuh_testing:monitoring.py:887 Add new file composer process for wazuh-master and path: /var/ossec/logs/cluster.log
DEBUG    wazuh_testing:monitoring.py:891 Add new file monitor process for wazuh-master and path: /var/ossec/logs/cluster.log
DEBUG    wazuh_testing:monitoring.py:993 Cleaning temporal files...
DEBUG    wazuh_testing:monitoring.py:982 Checking results...
----------------------------------------------------------------------------------------- Captured log teardown -----------------------------------------------------------------------------------------
DEBUG    testinfra:base.py:286 RUN CommandResult(command=b'ansible --tree /tmp/tmpvmx4sxe6 -i /home/yanazaeva/git/wazuh-qa/tests/system/provisioning/basic_cluster/inventory.yml -m replace --args \'path=/var/ossec/etc/ossec.conf regexp=\'"\'"\'</client>[\\s\\S]+<client_buffer>\'"\'"\' replace=\'"\'"\'</client>\n<client_buffer>\'"\'"\'\' wazuh-agent2', exit_status=0, stdout=b'wazuh-agent2 | CHANGED => {\n    "changed": true,\n    "msg": "1 replacements made"\n}\n', stderr=None)
INFO     testinfra:ansible.py:71 RUN Ansible('replace', "path=/var/ossec/etc/ossec.conf regexp='</client>[\\s\\S]+<client_buffer>' replace='</client>\n<client_buffer>'", {'check': False, 'become': False}): {'changed': True, 'msg': '1 replacements made'}
DEBUG    testinfra:base.py:286 RUN CommandResult(command=b"ansible --tree /tmp/tmpbcsf9bqx -i /home/yanazaeva/git/wazuh-qa/tests/system/provisioning/basic_cluster/inventory.yml -m command --args 'service wazuh-agent restart' wazuh-agent2", exit_status=0, stdout=b'wazuh-agent2 | CHANGED | rc=0 >>\nKilling wazuh-modulesd... \nKilling wazuh-logcollector... \nKilling wazuh-syscheckd... \nKilling wazuh-agentd... \nKilling wazuh-execd... \nWazuh v4.3.0 Stopped\nStarting Wazuh v4.3.0...\nStarted wazuh-execd...\nStarted wazuh-agentd...\nStarted wazuh-syscheckd...\nStarted wazuh-logcollector...\nStarted wazuh-modulesd...\nCompleted.\n', stderr=b"[WARNING]: Consider using the service module rather than running 'service'.  If\nyou need to use command because service is insufficient you can add 'warn:\nfalse' to this command task or set 'command_warnings=False' in ansible.cfg to\nget rid of this message.\n")
INFO     testinfra:ansible.py:71 RUN Ansible('command', 'service wazuh-agent restart', {'check': False, 'become': False}): {'changed': True,
 'cmd': ['service', 'wazuh-agent', 'restart'],
 'delta': '0:00:06.659630',
 'end': '2021-11-26 12:03:01.288718',
 'rc': 0,
 'start': '2021-11-26 12:02:54.629088',
 'stderr': '',
 'stderr_lines': [],
 'stdout': 'Killing wazuh-modulesd... \n'
           'Killing wazuh-logcollector... \n'
           'Killing wazuh-syscheckd... \n'
           'Killing wazuh-agentd... \n'
           'Killing wazuh-execd... \n'
           'Wazuh v4.3.0 Stopped\n'
           'Starting Wazuh v4.3.0...\n'
           'Started wazuh-execd...\n'
           'Started wazuh-agentd...\n'
           'Started wazuh-syscheckd...\n'
           'Started wazuh-logcollector...\n'
           'Started wazuh-modulesd...\n'
           'Completed.',
 'stdout_lines': ['Killing wazuh-modulesd... ',
                  'Killing wazuh-logcollector... ',
                  'Killing wazuh-syscheckd... ',
                  'Killing wazuh-agentd... ',
                  'Killing wazuh-execd... ',
                  'Wazuh v4.3.0 Stopped',
                  'Starting Wazuh v4.3.0...',
                  'Started wazuh-execd...',
                  'Started wazuh-agentd...',
                  'Started wazuh-syscheckd...',
                  'Started wazuh-logcollector...',
                  'Started wazuh-modulesd...',
                  'Completed.'],
 'warnings': ['Consider using the service module rather than running '
              "'service'.  If you need to use command because service is "
              "insufficient you can add 'warn: false' to this command task or "
              "set 'command_warnings=False' in ansible.cfg to get rid of this "
              'message.']}
======================================================================================== short test summary info ========================================================================================
FAILED tests/system/test_cluster/test_agent_info_sync/test_agent_info_sync.py::test_agent_info_sync - TimeoutError: Did not found the expected callback in wazuh-master: .*test_label.*
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! stopping after 1 failures !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
===================================================================================== 1 failed in 135.95s (0:02:15) =====================================================================================
```

This script is still going to test the system functioning properly, as some lines below it checks for the new information in the `global.db` database. 
https://github.com/wazuh/wazuh-qa/blob/158ab10a0dbbf0c9dcbd4cc2c718346bc8ba9c49/tests/system/test_cluster/test_agent_info_sync/test_agent_info_sync.py#L75-L84

--- a/tests/system/test_cluster/test_agent_info_sync/test_agent_info_sync.py
+++ b/tests/system/test_cluster/test_agent_info_sync/test_agent_info_sync.py
@@ -68,9 +68,6 @@ def test_agent_info_sync(clean_cluster_logs, remove_labels):
                                    replace=f'<labels><label key="{label}">value</label></labels>')
     host_manager.get_host(modified_agent).ansible('command', f'service wazuh-agent restart', check=False)
 
-    # Run the callback checks for the Master and Worker nodes
-    HostMonitor(inventory_path=inventory_path, messages_path=messages_path, tmp_path=tmp_path).run()
-
     # Check that the agent label is updated in the master's database.
     for i in range(10):
         if host_manager.run_command(
@@ -79,7 +76,7 @@ def test_agent_info_sync(clean_cluster_logs, remove_labels):
                 global_db_path,
                 "SELECT id FROM labels WHERE key='{}'".format(f'\\"{label}\\"'))):
             break
-        sleep(10)
+        sleep(21)
     else:
         pytest.fail(f"Label {label} couldn't be found in master's global.db database.")
 

